src/components/App.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+  <div id="app">
+    <template v-if="$store.state.user">
```

Comment:
Ok I assume "$store.state.user" is a variable managed by the Vuex store. It's marked true if the user is authenticated

Which means the v-if while only show this div if the user is authenticated